CHANGELOG.md

```diff
@@ -13,19 +13,24 @@ and includes an additional section for migration notes.
 - *Fixed* - Any bug fixes.
 - *Security* - Vulnerabilities fixes and changes.
 
-
 ## [Unreleased]
+
+## [6.0.0]
 ### Changed
 - *ORCA-290* Renamed `excludeFileTypes`, `orcaDefaultBucketOverride` and `orcaDefaultStorageClassOverride` to `excludedFileExtensions`, `defaultBucketOverride` and  `defaultStorageClassOverride` respectively. In addition, ORCA configuration variables `excludedFileExtensions`, `defaultBucketOverride` and `defaultStorageClassOverride` are now under `collection.meta.orca`.
 - *ORCA-290* Adjusted workflows/step functions for `OrcaRecoveryWorkflow`.
   - `excludeFileTypes`, `orcaDefaultBucketOverride` and `orcaDefaultStorageClassOverride` arguments in `task_config` are now `excludedFileExtensions`, `defaultBucketOverride` and  `defaultStorageClassOverride` respectively.
   - `excludedFileExtensions`, `defaultBucketOverride` and `defaultStorageClassOverride` keys are now under `collection.meta.orca`. See the example below under `Migration Notes`.
 - *ORCA-519* Enforced schema checks in `request_status_for_granule` and `request_status_for_job`.
   Both lambdas will return proper HTTP error codes for bad inputs of internal server errors.
+  Additionally, corrected error in [API Reference](https://nasa.github.io/cumulus-orca/docs/developer/api/orca-api) 
+  where the `error` status for these lambdas was incorrectly listed as `failed`.
+- *ORCA-320* Requests to API Gateway now use IAM permissions, restricting anonymous access.
 
 ### Migration Notes
 
-- Adjust workflows/step functions for `OrcaCopyToGlacierWorkflow`.
+- Adjust workflow/step-function usage for new key paths.
+  - Affects `OrcaCopyToGlacierWorkflow` and `OrcaRecoveryWorkflow`.
```

Comment:
I always think of migration notes as something that affects external users using the software. In this case, `OrcaCopyToGlacierWorkflow` and `OrcaRecoveryWorkflow` is something the ORCA team owns and manages. The changes really affect any step functions that the end user has that calls the `copy_to_glacier` task. I am wondering if we should reword this to be something like:
```
Adjust workflow/step-function usage for items that use `copy_to_glavier` task for new key paths.
```
Thoughts?

Reply:
Added separate line for CopyToGlacier usage, as I recall that some customers include copy_to_glacier in their own workflows.
Leaving the other section for now, as anyone integrating with our workflows still needs to know about the key changes.